metals/src/main/scala/scala/meta/internal/metals/MetalsLogger.scala

```diff
@@ -88,21 +89,8 @@ object MetalsLogger {
     }
   }
 
-  def newBspLogger(workspace: AbsolutePath): Logger = {
```

Comment:
This doesn't seem to be used anywhere. I thought it may have been for the bsp trace related stuff, but that's taken care of in the `GlobalLogger`.